src/store.ts

```diff
@@ -29,6 +29,7 @@ export interface GraphState {
   centerPosition?: CenterPositionVector;
   actives?: string[];
   selections?: string[];
+  hoveredNodeId?: string;
```

Comment:
Whats this for ?

Reply:
If node is hovered than dragging in cluster is disabling

Reply:
I think this needs to be multi - what if you lasso them?

Reply:
Let's also add comments for all these methods ( i know u didn't write them all but need to add them ).

Reply:
We don't need it as multi because hovered node can be only one node which under mouse cursor.
HoveredNodeId property uses only for detect which element should be dragged **node** or **cluster** (if hovered node is present than dragging for cluster is disabling)
Lasso is uses for selected state.

Reply:
Still need to add comments

Reply:
added